Source/Particles/WarpXParticleContainer.H

```diff
@@ -531,7 +531,7 @@ public:
     //amrex::Real getCharge () {return charge;}
     amrex::ParticleReal getCharge () const {return charge;}
     //amrex::Real getMass () {return mass;}
```

Comment:
```suggestion
    //amrex::Real getMass () {return m_mass;}
```
I think that we should do that for consistency. However, I also think that this line should be removed! Is there a reason to keep this commented out method ?

Reply:
Thanks Luca, I had not noticed that. I removed the two methods that were commented out, `getMass` and `getCharge`. I don't see any reason why they should be kept there as is, especially given that there is no comment explaining what they are there for.